app/services/stats/etl/offender_sar_complaint_closed_cases.rb

```diff
@@ -0,0 +1,59 @@
+require 'csv'
+
+module Stats
+  module ETL
+    class OffenderSarComplaintClosedCases < BaseClosedCases
+
+      RESULT_NAME = 'offender-sar-complaint-closed-cases'.freeze
+
+      CSV_COLUMN_HEADINGS = [
+        'Case number',
+        'Data subject name',
+        'Subject type',
+        'Who is making the request?',
+        'Company name',
+        'Date received at MOJ',
+        'Date case closed',
+        'Timeliness (in time/out of time)',
+        'No of days taken',
+        'Pages for dispatch',
+        'Exempt pages',
+        'Final page count'
+      ]
+
+      FIELD_COLUMNS = [
+        'number',
+        'sar_subject_full_name',
+        'sar_subject_type',
+        'requester_type',
+        'third_party_company_name',
+        'date_received',
+        'date_responded',
+        " case when number_of_days_late > 0 then 'out of time' else 'in time' end ",
+        'number_of_days_taken',
+        'number_of_final_pages::integer - number_of_exempt_pages::integer',
+        'number_of_exempt_pages',
+        'number_of_final_pages'
+      ]
+
+      def result_name
+        RESULT_NAME
+      end
+
+      private
+
+      def columns
+        @_columns ||= begin
+          OffenderSarComplaintClosedCases::FIELD_COLUMNS
+        end
+      end
+
+      def heading
+        @_heading ||= begin
+          CSV.generate_line(OffenderSarComplaintClosedCases::CSV_COLUMN_HEADINGS, force_quotes: true)
+        end
+      end
+
+    end
+  end
+end
```

Comment:
If this is just a copy of the OffenderSarClosedCases, I would suggest to simply inherit from OffenderSarClosedCases during cloning stage. 
`class OffenderSarComplaintClosedCases < OffenderSarClosedCases`